example/properties_directory/structural/point_group_symmetry.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "point_group_symmetry",
```

Comment:
to keep it clear and easier to find, let's call it `symmetry_symbol_point_group`, `symmetry_symbol_space_group`?